microservice_utils/novu/adapters.py

```diff
@@ -10,12 +10,18 @@ def __init__(self, api_key, base_url="https://api.novu.co"):
         self.event_api = EventApi(base_url, api_key)
 
     def send_notification(
-        self, name, users: list[UUID], context: dict[str, typing.Any], **kwargs
+        self,
+        name,
+        users: list[UUID],
+        context: dict[str, typing.Any],
+        overrides: dict[str, typing.Any],
```

Comment:
Wrap `dict[str, typing.Any]` with `typing.Optional[]`. That way it will accept None as a value for overrides:
`overrides: typing.Optional[dict[str, typing.Any]],`

Have it default to None so that existing implementations of `send_notification` don't need to be updated:
`overrides: typing.Optional[dict[str, typing.Any]] = None,`